GliaWidgets/RemoteConfiguration/RemoteConfiguration.swift

```diff
@@ -33,6 +34,24 @@ public extension RemoteConfiguration {
         }
     }
 
+    struct Alert: Codable {
+        let titleFont: Font?
```

Comment:
I guess `titleFont` and `titleColor` can be replaced with `title: Text?`
And the same for `message`

Reply:
Fixed.